### ISSUE TITLE
modwording: while focussed

### ISSUE THREAD

**Issue by ot4ku92** [Openarl/PathOfBuilding#1360]
Hey. Thanks a lot for PoB. I noticed that the wording for the mod "x% chance to deal double damage while focussed" doesn't work in PoB - it works if I remove one s and make it "focused". Might also be wrong on other similar mods, haven't tested, just wanted to let you know.

**Comment by DaCurse** [comment]
yup. both are correct gramatically but pob and the game have them different. example:
![Example item in-game](https://cdn.discordapp.com/attachments/175290321695932416/533769842722078730/unknown.png)
![Example item in PoB](https://cdn.discordapp.com/attachments/175290321695932416/533769904361570314/unknown.png)
![Example item in PoB after changing the word](https://cdn.discordapp.com/attachments/175290321695932416/533769966202388491/unknown.png)

**Comment by aydjay** [comment]
Looks like vaal pact isn't being applied to the amulet mod:

You have Vaal Pact while Focussed
10% of Damage Leeched as Life while Focussed

Completely cut down POB tree generated by version 1.4.129: https://pastebin.com/GDmGJvY7
